tests/unit/test_sql_analyser.py

```diff
@@ -13,14 +13,22 @@ def analyse_sql(self, sql):
             sql_statements=sql, database_vendor=self.database_vendor
         )
 
-    def assertValidSql(self, sql):
-        errors, _ = self.analyse_sql(sql)
+    def assertValidSql(self, sql, allow_warnings=False):
+        errors, _, warnings = self.analyse_sql(sql)
         self.assertEqual(0, len(errors), "Found errors in sql: {}".format(errors))
+        if not allow_warnings:
+            self.assertEqual(
+                0, len(warnings), "Found warnings in sql: {}".format(errors)
+            )
 
     def assertBackwardIncompatibleSql(self, sql):
-        errors, _ = self.analyse_sql(sql)
+        errors, _, _ = self.analyse_sql(sql)
```

Comment:
NIP: or even shorter:

```suggestion
        errors, *_ = self.analyse_sql(sql)
```

Reply:
We eventually will want to refactor the return to container object that will hold all fields.
However, I do prefer this failing for now when we change the number of returned arguments